patches/extras/0002-libjsonpbparse-Fix-build-after-protobuf-util-Status-.patch

```diff
@@ -0,0 +1,32 @@
+From 3924f7db9b70f83e88062ea6f481e458293977d6 Mon Sep 17 00:00:00 2001
+From: MoetaYuko <loli@yuko.moe>
+Date: Fri, 10 Dec 2021 17:36:31 +0800
+Subject: [PATCH] libjsonpbparse: Fix build after protobuf util::Status
+ refactorization
+
+---
+ libjsonpb/parse/jsonpb.cpp | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/libjsonpb/parse/jsonpb.cpp b/libjsonpb/parse/jsonpb.cpp
+index 3a042e7..e65a39d 100644
+--- a/libjsonpb/parse/jsonpb.cpp
++++ b/libjsonpb/parse/jsonpb.cpp
+@@ -48,7 +48,7 @@ ErrorOr<std::string> MessageToJsonString(const Message& message) {
+                                    &json, options);
+ 
+   if (!status.ok()) {
+-    return MakeError<std::string>(status.error_message().as_string());
++    return MakeError<std::string>(status.message().as_string());
```

Comment:
I'm not sure if this needs to be ifdef'd. https://github.com/protocolbuffers/protobuf/pull/8354 is the breaking change.